packages/libfaac/build.sh

```diff
@@ -0,0 +1,10 @@
+TERMUX_PKG_HOMEPAGE=http://faac.sourceforge.net/
+TERMUX_PKG_DESCRIPTION="Freeware Advanced Audio Coder AAC Enoder"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_LICENSE=GPL-2.0
+TERMUX_PKG_LICENSE_FILE="COPYING"
+TERMUX_PKG_VERSION=1.29.9.2
+_VERSION_REAL=$(cut -f1,2 -d '.' <<< ${TERMUX_PKG_VERSION})
```

Comment:
Use slices. The cut command is overkill there.

Don't copy code snippets unless you understand why they needed. https://github.com/termux/termux-packages/pull/8174#discussion_r766960647